styles.css

```diff
@@ -200,49 +218,29 @@ img {
 }
 
 
-.idea-grid{
-  /* display: grid;
-  border-style: solid; */
+.idea-grid {
```

Comment:
Curious to see how this will look. If it still gives us trouble I think we could try flex instead possibly?